### ISSUE TITLE
MirthAPI' object has no attribute 'get_channels'

### ISSUE THREAD

**Issue by yooper** [renalreg/python-mirth-client#50]
I am getting the following error when trying to run the demo code. 
```
>               for channel in await api.get_channels():
E               AttributeError: 'MirthAPI' object has no attribute 'get_channels'
```

Here is the code I am using
```python            
async def get_mirth_api_connect(settings: Settings) -> MirthAPI:
    """ Returns a MirthApi connection before authentication. """
    return MirthAPI(settings.mirth_api_connect_url)            
            
async def test_mirth_connect_login_success() -> None:
    settings = Settings()
    async with await get_mirth_api_connect(settings) as api:
        await api.login(settings.mirth_api_connect_user_name, settings.mirth_api_connect_user_password)
        # Check out list of channels
        for channel in await api.get_channels():
            metadata = await channel.get()
            print(f"ID: {metadata.id}")
            print(f"Name: {metadata.name}")
            print("")
```

**Comment by jtc42** [comment]
Thanks for flagging this. I forgot to update the README example after some old functions were deprecated. I'll get this corrected ASAP.

In the meantime, the `get_channels()` method can be replaced with `channel_info()`

**Comment by yooper** [comment]
Here is the code I ended up using to get it working

```python
        for channel in await api.channels():
            metadata = await channel.get_info()
            print(f"ID: {metadata.id}")
            print(f"Name: {metadata.name}")
            print("")

```

Thank you for creating this library.

**Comment by jtc42** [comment]
Thanks! Resolved by 7617c3ad5accd543dffcbc542b6ebf345cea9c06
